flathunter/config.py

```diff
@@ -354,6 +349,23 @@ def max_price_per_square(self):
         """Return the configured maximum price per square meter"""
         return self._get_filter_config("max_price_per_square")
 
+    def max_distance(self) -> List[DistanceConfig] | None:
```

Comment:
I wonder if it would be more natural here just to return the empty list of there's nothing configured. Makes the typing judgement simpler.